src/components/KnownFollowers.tsx

```diff
@@ -0,0 +1,154 @@
+import React from 'react'
+import {View} from 'react-native'
+import {AppBskyActorDefs, moderateProfile, ModerationOpts} from '@atproto/api'
+import {msg, plural, Trans} from '@lingui/macro'
+import {useLingui} from '@lingui/react'
+
+import {makeProfileLink} from '#/lib/routes/links'
+import {isNative} from '#/platform/detection'
+import {sanitizeDisplayName} from 'lib/strings/display-names'
+import {UserAvatar} from '#/view/com/util/UserAvatar'
+import {atoms as a, useTheme} from '#/alf'
+import {Link} from '#/components/Link'
+import {Text} from '#/components/Typography'
+
+const AVI_SIZE = 26
+const AVI_BORDER = isNative ? 1 : 1
+
+export function KnownFollowers({
+  profile,
+  moderationOpts,
+}: {
+  profile: AppBskyActorDefs.ProfileViewDetailed
+  moderationOpts: ModerationOpts
+}) {
+  if (!profile.viewer?.knownFollowers) {
+    return null
+  }
```

Comment:
Note to self, check for `count === 0` here too

Reply:
Actually, we're going to check for `followers.length` instead, since blocks will still count towards `count`, see https://github.com/bluesky-social/atproto/pull/2557/files#r1634012991